<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfileAnalisisWidget</class>
 <widget class="QMainWindow" name="ProfileAnalisisWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1045</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="2">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCustomPlot" name="plotRMR" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>250</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="Line" name="line">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QCustomPlot" name="plotMain" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>250</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="2">
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>ТРАССЫ</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QTreeWidget" name="tracesList">
           <property name="focusPolicy">
            <enum>Qt::StrongFocus</enum>
           </property>
           <property name="headerHidden">
            <bool>true</bool>
           </property>
           <property name="columnCount">
            <number>5</number>
           </property>
           <attribute name="headerVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="headerCascadingSectionResizes">
            <bool>true</bool>
           </attribute>
           <attribute name="headerDefaultSectionSize">
            <number>200</number>
           </attribute>
           <column>
            <property name="text">
             <string>Параметр</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Значение</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>RMR</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>dMX</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>dMY</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>УПРАВЛЕНИЕ</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="DeviceControlWidget" name="deviceControlWidget" native="true"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCustomPlot" name="plotHist" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>250</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionReset"/>
   <addaction name="actionPAN"/>
   <addaction name="actionPTS"/>
   <addaction name="actionRLine"/>
   <addaction name="actionRFit"/>
   <addaction name="actionSelectionZoom"/>
   <addaction name="actionMarkers"/>
   <addaction name="actionCut"/>
   <addaction name="separator"/>
   <addaction name="actionToMeasurement"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionToMeasurement">
   <property name="text">
    <string>В ИЗМЕРЕНИЯ</string>
   </property>
   <property name="toolTip">
    <string>Открыть трассу для измерений</string>
   </property>
  </action>
  <action name="actionPAN">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/pan.png</normaloff>:/icons/icons/pan.png</iconset>
   </property>
   <property name="text">
    <string>Перемещение</string>
   </property>
   <property name="toolTip">
    <string>Перемещение</string>
   </property>
  </action>
  <action name="actionPTS">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/pts.png</normaloff>:/icons/icons/pts.png</iconset>
   </property>
   <property name="text">
    <string>PTS</string>
   </property>
   <property name="toolTip">
    <string>Показать точки</string>
   </property>
  </action>
  <action name="actionRLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/rline.png</normaloff>:/icons/icons/rline.png</iconset>
   </property>
   <property name="text">
    <string>R-LN</string>
   </property>
   <property name="toolTip">
    <string>R-линия</string>
   </property>
  </action>
  <action name="actionRFit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/rline_s.png</normaloff>:/icons/icons/rline_s.png</iconset>
   </property>
   <property name="text">
    <string>R-FT</string>
   </property>
   <property name="toolTip">
    <string>Спрямить по R-линии</string>
   </property>
  </action>
  <action name="actionSelectionZoom">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/select_z.png</normaloff>:/icons/icons/select_z.png</iconset>
   </property>
   <property name="text">
    <string>| + |</string>
   </property>
   <property name="toolTip">
    <string>Приближение выделения</string>
   </property>
  </action>
  <action name="actionMarkers">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/select.png</normaloff>:/icons/icons/select.png</iconset>
   </property>
   <property name="text">
    <string>| S |</string>
   </property>
   <property name="toolTip">
    <string>Выделение</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/cutsel.png</normaloff>:/icons/icons/cutsel.png</iconset>
   </property>
   <property name="text">
    <string>| C |</string>
   </property>
   <property name="toolTip">
    <string>Вырезать выделение</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/reset.png</normaloff>:/icons/icons/reset.png</iconset>
   </property>
   <property name="text">
    <string>RST</string>
   </property>
   <property name="toolTip">
    <string>Сбросить</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">../qcustomplot/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DeviceControlWidget</class>
   <extends>QWidget</extends>
   <header>device/devicecontrolwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
